doc/nrf/ug_matter_gs_testing.rst

```diff
@@ -8,11 +9,18 @@ When you build any of the available Matter samples to the supported development
 The development kit and the application running Matter stack that is programmed on the development kit together form the Matter accessory device.
 
 The |NCS| supports Matter stack that is built on top of a low-power, 802.15.4-compatible Thread network, or on top of a Wi-Fi network.
-To commission and control the Matter accessory device remotely over either of these networks, you need to set up a Matter controller on a mobile or a PC.
+To control the Matter accessory device remotely over either of these networks, you need to set up a Matter controller on a mobile or a PC.
 Matter controller is a role within the Matter environment.
 This controller interacts with the accessory devices using the following protocols:
 
-* Bluetooth® LE during the commissioning process - to securely pass the network credentials and provision the accessory device into the Thread network.
+* Bluetooth® LE during the commissioning process - to securely pass the network credentials and provision the accessory device into the Thread network during commissioning.
+  At this stage, the controller has the commissioner role.
+
+  To start the commissioning procedure, the controller must get the onboarding information from the Matter accessory device.
+  The data payload, which includes the device discriminator and setup PIN code, is encoded within a QR code, printed to the UART console, and can be shared using an NFC tag.
```

Comment:
```suggestion
  The data payload, which includes the device discriminator and setup passcode, is encoded within a QR code, printed to the UART console, and can be shared using an NFC tag.
```

Reply:
I've actually removed this line and kept it only in the Commissioning section.